.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Usually, you cannot use comments in `.json` files. I think it's better if you remove them.

Reply:
It was generated by VSCode directly, so I let them. But yeah I can remove them.